en/docs/observability/perform-root-cause-analysis.md

```diff
@@ -8,88 +8,94 @@ Performing root cause analysis is crucial in identifying and rectifying the unde
     
-## Detecting anomalies and performing root cause analysis
-You can detect the anomalies of the usage of a service by observing the **Throughput & Latency** graphs. The throughput graph shows the successful and erroneous requests that occurred during a period. The latency graph shows the latency of each request. You can observe these graphs at a lower granularity by choosing a shorter time range, which gives you a better understanding of the incidents.
+## Detect anomalies and perform root cause analysis
+You can detect the anomalies of the usage of a service by observing the **Throughput & Latency** graphs. The throughput graph shows the successful and erroneous requests that occurred during a period. The latency graph shows the latencies of each request. You can observe these graphs at a lower granularity by choosing a shorter time range, which gives you a better understanding of the incidents.
 
-### Analyzing the throughput graph
-The throughput graph will show the throughput of requests per hour for a selected timestamp. 
+### Step 1: Detect the anomalies
 
-With the sample data, you will observe spikes in the throughput graph. Let's see how you can find
-the root cause by analyzing this data.
+#### Analyze the throughput graph
+The throughput graph will show the throughputs of requests per hour for a selected timestamp. 
+
+With the sample data, you will observe spikes in the throughput graph.  You can find the root cause for this by learning how to analyze the throughput graph.
 
 Follow the steps below. 
 
-1. Hover over the throughput graph at 01/04/2021 13:41:00. You can observe that 46.06 errors had occurred at the time. 
+1. Hover over the throughput graph where you see spikes. You can observe the number of errors that occurred at the time. 
     
     !!!tip
         You can expand the graph by clicking and dragging the cursor over the period you want to drill down.
         
-2. Click on the graph on 01/04/2021 at 13:41:00.
+2. Click the graph on a high point.
 3. Observe the logs in the logs panel.
 4. Note multiple logs are mentioning a connection error with the hr-service.
  
 ![Analyzing the throughput graph](../assets/img/observability/throughput-graph-analysis.png)
  
-### Analyzing the latency graph
+#### Analyze the latency graph
 
 The latency graph shows the latencies of each request.
 
-With the sample data, you will observe spikes in the latency graph. By analyzing the latency graph, you can identify which API invocation caused the error.  Let's see how to find the
-root cause by analyzing this data.
+With the sample data, you will observe spikes in the latency graph. By analyzing the latency graph, you can identify the errors that caused the spikes and the exact API invocation(s) that caused the errors. 
 ![Analyzing the latency graph](../assets/img/observability/latency-graph-analysis.png)
 
-Follow the steps below.
+Follow the steps below to learn how to analyze the latency graph.
```

Comment:
Why do we need to add `to learn how to analyze the latency graph` part? If we need it let's add it for `Analyze the throughput graph` section as well.

Reply:
This was originally worded to have a flow when read after the sentences above.
Reworded.